autode/bracket/imagepair.py

```diff
@@ -523,17 +545,17 @@ def run_cineb_from_end_points(self) -> None:
         return None
 
     @property
-    def _total_history(self) -> OptimiserHistory:
+    def _total_history(self) -> Iterator[CartesianCoordinates]:
         """
         The total history of the image-pair, including any CI run
         from the endpoints
         """
-        history = OptimiserHistory()
-        history.extend(self._left_history)
+        cineb_coords = []
         if self._cineb_coords is not None:
-            history.append(self._cineb_coords)
-        history.extend(self._right_history[::-1])  # reverse order
-        return history
+            cineb_coords.append(self._cineb_coords)
```

Comment:
I think it might be okay to remove the CINEB part entirely, I put that in on advice from another colleague who is also working in similar methods. The problem was that one of the images would fall over the barrier, so we stop earlier, and then run a CINEB to get close to the TS. I think that much easier and faster method would be to do cubic interpolation and take the highest point on the interpolated line. (I didn't make any changes here as I am trying to keep PR's small)